cylc/uiserver/app.py

```diff
@@ -126,13 +129,17 @@ class CylcUIServer(ExtensionApp):
 
     name = 'cylc'
     app_name = 'cylc-gui'
-    default_url = "/cylc"
     load_other_extensions = True
     description = '''
-    Cylc - A user interface for monitoring and controlling Cylc workflows.
+    Cylc gui - A user interface for monitoring and controlling Cylc workflows.
     '''  # type: ignore[assignment]
     examples = '''
-        cylc gui    # start the cylc GUI
+        cylc gui                  # Start the Cylc GUI (At the dashboard page)
+cylc gui [workflow]       # Start the Cylc GUI (at the workflow page),
+cylc gui --new [workflow] # Start the Cylc GUI (at the workflow page), with a
+                            new instance. By default, if there is an existing
+                            gui instance, Cylc will use that.
+
     '''  # type: ignore[assignment]
```

Comment:
I'll change the wording. The formatting is dealt with through traitlets and they indent and dedent it with `yield indent(dedent(self.examples.strip()))`. An additional dedent as suggested is having no effect for me.

Reply:
From my testing the `dedent` is needed to avoid it being like this:
```
   examples = '''
        cylc gui                  # ...
cylc gui [workflow]       # ...
```

Reply:
That is really strange. Two screengrabs from me below with/without dedent:
![image](https://user-images.githubusercontent.com/37735232/197184063-82c7fff5-62a5-4558-bc35-de3803a8e98e.png)

![image](https://user-images.githubusercontent.com/37735232/197184541-70cf53bc-0a5f-489d-a1b5-0e2d66997d8c.png)

Reply:
Incidentally, the `If there is an existing GUI instance, Cylc will use that.` is required for both the `cylc gui` and the `cylc gui [workflow]` options. I actually think it is neater leaving it as it is, since it conveys the same information without repetition, what do you think?